### PR TITLE
added color palette and folder structure

### PR DESCRIPTION
**Pull request by pdabrowsky**
https://app.asana.com/0/1202705605335915/1202705556342442/f

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   settings: {
     "import/resolver": {
       node: {
-        extensions: [".ts", ".tsx"],
+        extensions: [".scss", ".ts", ".tsx"],
```
Comment (dominikx96) [inline .eslintrc.js:6]: Eslint doesn't lint your styles, you don't need the `.scss` extension here

File: src/Styles/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+//color palette
```
Comment (dominikx96) [inline src/Styles/_variables.scss:1]: - I don't think we need a comment here, it's obvious it's a color palette, doesn't give us any value
- make `styles` directory lowercase

File: src/Styles/_variables.scss [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (dominikx96) [inline src/Styles/_variables.scss:2]: I think we have more colors like gray and light black

File: src/Styles/index.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
+body {
```
Comment (dominikx96) [inline src/Styles/index.scss:1]: - You should import the variables file here and then import the `index.scss` in the `App` component
-

File: src/Styles/index.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+      sans-serif;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+  }
+  
+  code {
+    font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+      monospace;
+  }
+
```
Comment (dominikx96) [inline src/Styles/index.scss:14]: Move these default styles into `_base.scss` and import that in the `index.scss`

**Review by dominikx96 — CHANGES_REQUESTED** [review]

File: src/Styles/_variables.scss [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+$gold: #C3A275;
+
+$dark: rgba(18, 18, 18, 0.7);
+$lightBlack: rgba(0, 0, 0, 0.2);
+$strongBlack: rgba(0, 0, 0, 0.6);
+
+$grey: rgba(255, 255, 255, 0.8);
+$darkGrey: rgba(255, 255, 255, 0.7);
+
+$gradient: linear-gradient(0deg, rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7));
```
Comment (dominikx96) [inline src/Styles/_variables.scss:12]: What about a situation when you'll have more than one gradient? Update this name :)

**Review by dominikx96 — CHANGES_REQUESTED** [review]
This code doesn't work for me. Please fix errors, you should request a review once you test your changes. 

Follow docs to resolve a problem with sass: https://create-react-app.dev/docs/adding-a-sass-stylesheet

Screenshot 2022-08-08 at 09 50 59

**Review by dominikx96 — CHANGES_REQUESTED** [review]
I assume you haven't tested that:
Screenshot 2022-08-09 at 18 32 37

Please test before requesting a review:
- Add a text into the `App` component
- Style that text to be in $gold color

**Comment by pdabrowsky** [comment]
I tested it, i think it works. It didn't work for you because you didn't specify the source file.

**Review by dominikx96 — APPROVED** [review]

**Pull request merged**